commands/template_store_list.go

```diff
@@ -0,0 +1,170 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// The URL where the store resides
+	DefaultTemplateList = "https://raw.githubusercontent.com/martindekov/store/martindekov/unofficial_templates_json/templates.json"
+	incubatorRepo       = "openfaas-incubator"
```

Comment:
Why do we have this?

Reply:
it names the source of our templates, because including them all every single time might be little bit spammy